commands/project_sync.go

```diff
@@ -283,7 +286,7 @@ func (cmd *ProjectSync) LoadComposeFile() (*ComposeFile, error) {
 // when compiled without -cgo this executable will not use the native mac dns resolution
 // which is how we have configured dnsdock to provide names for containers.
 func (cmd *ProjectSync) WaitForUnisonContainer(containerName string, timeoutSeconds int) (string, error) {
-	cmd.out.Info("Waiting for container to start")
+	cmd.out.SpinWithVerbose("Sync container '%s' started , waiting for unison server process...", containerName)
```

Comment:
I never see this message.

Reply:
Ignore this, I wasn't supposed to because I was running in non verbose mode. It's there in verbose mode.